test/LibraryManager.IntegrationTest/AutoCompletionTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Threading;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AutoCompletionTests : VisualStudioLibmanHostTest
```

Comment:
nit: these are Completion tests but not AutoCompletion (which would be where it inserts the only matching item)

Reply:
I see. I will use a different name. LibmanCompletionTests probably? I will add some other completion deals with libman.json there.